schema/crdb/dbinit.sql

```diff
@@ -4734,6 +4737,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.bp_omicron_zone (
     image_source omicron.public.bp_zone_image_source NOT NULL,
     image_artifact_sha256 STRING(64),
 
+    -- Generation for Nexus zones
+    nexus_generation INT8,
```

Comment:
Could we add a check constraint that this is `NOT NULL` for Nexus zones and `NULL` otherwise?

Reply:
Added in https://github.com/oxidecomputer/omicron/pull/8944/commits/eafafe4a8f6dfa4c5e7fa436e675f5956aa4363e